plugins/docker_binary_builder/README.md

```diff
@@ -0,0 +1,63 @@
+# Docker Builder for Binaries
+
+This plugin is useful when your projects are based on compiled languages and you don't want to base
+your deployment docker images on images that have all the tools/libs/compilers installed.
+
+Advantages to separating your docker images include:
+- Smaller deployment docker images.
+- Better security in your docker images to not be able to recompile code.
+
+## Getting Started
+
+This plugin makes a few assumptions:
+- You have 2 dockerfiles:
+ - 'Dockerfile' used to generate your main deployment docker image
+ - 'Dockerfile.build' used to create a temporary image to build any binaries you want to package into the former.
+- You have a build script 'build.sh' that can be executed to compile the needed binaries
+  - This is packaged inside the Dockerfile.build image and executed
```

Comment:
We originally designed it that way to have the CMD inside the dockerfile, but the docker-api wasn't telling us when the cmd finished or the exit code.
This might have been user-error, but we gave it a few tries. I can have another look now that I'm much more familiar with the library.